.github/workflows/branch-changes-release.yaml

```diff
@@ -0,0 +1,63 @@
+name: Release Branch Changes
+on:
+  pull_request:
+    types: [labeled, synchronize] # Trigger on label and push events in pull requests
+
+jobs:
+  release-changes:
+    if: |
+      github.event.label.name == 'release-changes' || (
+        github.event_name == 'pull_request' &&
+        contains(github.event.pull_request.labels.*.name, 'release-changes')
+      )
+    timeout-minutes: 3
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Generate channel name
+        id: generate-channel-info
+        shell: bash
+        run: |
+          GITHUB_BRANCH_NAME=${GITHUB_HEAD_REF}
+
+          # Cleaning refs/*/
+          # refs/whatever/main => main
+          export GITHUB_BRANCH_NAME=${GITHUB_BRANCH_NAME#refs/[a-z]*/}
+
+          # Normalize image tags characters
+          # my-branch/test-01 => my-branch_test-01
+          export NORMALIZED_GIT_BRANCH_NAME=${GITHUB_BRANCH_NAME//[^[:alnum:]]/_}
+
+          # To lowercase all characters
+          export NORMALIZED_GIT_BRANCH_NAME=${NORMALIZED_GIT_BRANCH_NAME,,}
+
+          # Set maximum 128 characters for image name
+          export NORMALIZED_GIT_BRANCH_NAME=${NORMALIZED_GIT_BRANCH_NAME:0:127}
+
+          # Share the variable for further steps
+          echo "channel-name=${NORMALIZED_GIT_BRANCH_NAME}" >> "$GITHUB_OUTPUT"
```

Comment:
do you know if replicated has a max length for the channel name?

Reply:
No 🙇🏼 I added that limit as it's the same we're using in cloud-native for the docker images and looked okay for me just in case 😄